src/runoff.c

```diff
@@ -194,7 +194,13 @@ void runoff_execute()
     IsRaining = FALSE;
     for (j = 0; j < Nobjects[GAGE]; j++)
     {
-        gage_setState(j, currentDate);
+	// Check if rain is input though external data source (rainAPI)
+	if (Gage[j].external_rain == 0)
+	{
+        	gage_setState(j, currentDate);
+	}
+	// TODO Ensure check that nothing else happnes !
```

Comment:
It is a little bit unclear tp the reader how we are keeping existing functionality (using EPA input file to pull in TS in external file or pull in TS in input file) while adding the new functionality (adding TS from external script injections, adding TS from external file through other mechanism (e.g. external script).